landing-page/landing-page.js

```diff
@@ -0,0 +1,16 @@
+let rightArrow = document.querySelector(`.right-arrow`);
+let leftArrow = document.querySelector(`.left-arrow`);
+
+console.log(rightArrow);
+
+rightArrow.addEventListener(`click`, goToMusicPage);
+
+leftArrow.addEventListener(`click`, goToGamePage);
+
+function goToMusicPage () {
+  window.open(`./home-music/home-music.html`, `_self`);
+}
+
+function goToGamePage () {
+  window.open(`./home-game/home-game.html`, `_self`);
+}
```

Comment:
Since you want to open a new page in the same window, it is preferable to use `location.href` property  instead of `open` function and pass another parameter `_self`

> You can omit window

```suggestion
function goToMusicPage () {
  location.href = './home-music/home-music.html'
}

function goToGamePage () {
  location.href = './home-game/home-game.html'
}
```